t/00-compile-check-all.t

```diff
@@ -18,6 +18,10 @@ use strict;
 use warnings;
 use Test::Compile;
 use Test::Warnings;
+# We need :no_end_test here because otherwise it would output a no warnings
+# test for each of the modules, but with the same test number
+use Test::Warnings ':no_end_test';
```

Comment:
ok, but should we really include Test::Warnings twice?

Reply:
oh, my mistake! fixed